examples/Billing/AddBillingSetup.cs

```diff
@@ -0,0 +1,205 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V3.Errors;
+using Google.Ads.GoogleAds.V3.Resources;
+using Google.Ads.GoogleAds.V3.Services;
+using Google.Api.Gax;
+using System;
+using System.Linq;
+using static Google.Ads.GoogleAds.V3.Enums.TimeTypeEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V3
+{
+    /// <summary>
+    /// This example creates a billing setup for a customer. A billing setup is a link between
```

Comment:
I think we need to clarify that this code example illustrate how we can create a new Billing Setup reusing an existing Payments Account or creating a new Payments Account with a given Payments Profile.

Reply:
Added clarification.